docs/en/Running-Presto-with-Alluxio.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: global
+title: Running Facebook Presto with Alluxio
+nickname: Presto
+group: Frameworks
+priority: 2
+---
+
+This guide describes how to run [Facebook Presto](https://prestodb.io/) with Alluxio, so
+that you can easily use presto to query Hive tables in Alluxio's tiered storage.
+
+# Prerequisites
+
+The prerequisite for this part is that you have [Java](Java-Setup.html). And the Java version must greater then
+1.8. Alluxio cluster should also be set up in accordance to these guides for either
+[Local Mode](Running-Alluxio-Locally.html) or [Cluster Mode](Running-Alluxio-on-a-Cluster.html).
+
+Please[Download Presto](https://repo1.maven.org/maven2/com/facebook/presto/presto-server/)。And have finished
+[Hive On Alluxio](http://www.alluxio.org/docs/master/cn/Running-Hive-with-Alluxio.html)
```

Comment:
I think the link should just be "Running-Hive-with-Alluxio.html", so that it doesn't always point to the "cn" version.